package.json

```diff
@@ -54,8 +55,10 @@
     "eslint": "^7.3.1",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-vue": "^6.2.2",
+    "http-server": "^0.12.3",
     "node-sass": "4.14.1",
     "prettier": "^2.0.5",
+    "rimraf": "^3.0.2",
```

Comment:
what is this for?

Reply:
this is like a node alias for `rm -rf` command. performance testing is best done in production, so I added a local http-server to  host `dist` folder. however I stumbled upon some caching issues, so decided to remove dist folder each time before running build command